<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopWindow</class>
 <widget class="QMainWindow" name="TopWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="headerText">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="enterText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="gameManager"/>
    <addaction name="openSaveDontReload"/>
    <addaction name="saveGame"/>
    <addaction name="startGameAgain"/>
    <addaction name="undo"/>
    <addaction name="redo"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="searchMenu">
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="title">
     <string>Просмотр</string>
    </property>
   </widget>
   <widget class="QMenu" name="soundMenu">
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="title">
     <string>Звук</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="searchMenu"/>
   <addaction name="soundMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="gameManager">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Менеджер игр</string>
   </property>
  </action>
  <action name="addGame">
   <property name="text">
    <string>Добавить игру в библиотеку</string>
   </property>
  </action>
  <action name="openSave">
   <property name="text">
    <string>Открыть сохранение</string>
   </property>
  </action>
  <action name="openSaveDontReload">
   <property name="text">
    <string>Открыть сохранение без перезагрузки библиотеки</string>
   </property>
  </action>
  <action name="saveGame">
   <property name="text">
    <string>Сохранить игру</string>
   </property>
  </action>
  <action name="startGameAgain">
   <property name="text">
    <string>Начать игру заново</string>
   </property>
  </action>
  <action name="resetAllSettings">
   <property name="text">
    <string>Сбросить все настройки</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="setting">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="search">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="searchFurther">
   <property name="text">
    <string>Найти далее</string>
   </property>
  </action>
  <action name="sceneText">
   <property name="text">
    <string>Текст сцены</string>
   </property>
  </action>
  <action name="objectList">
   <property name="text">
    <string>Перейти к списку объектов</string>
   </property>
  </action>
  <action name="inventList">
   <property name="text">
    <string>Перейти к списку инвентаря</string>
   </property>
  </action>
  <action name="wayList">
   <property name="text">
    <string>Перейти к списку путей</string>
   </property>
  </action>
  <action name="debugMode">
   <property name="text">
    <string>Режим отладки</string>
   </property>
  </action>
  <action name="logOn">
   <property name="text">
    <string>Включить логирование</string>
   </property>
  </action>
  <action name="musicUp">
   <property name="text">
    <string>Музыка громче</string>
   </property>
  </action>
  <action name="musicLow">
   <property name="text">
    <string>Музыка тише</string>
   </property>
  </action>
  <action name="switchMusic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Музыка включена</string>
   </property>
  </action>
  <action name="soundUp">
   <property name="text">
    <string>Звуки громче</string>
   </property>
  </action>
  <action name="soundLow">
   <property name="text">
    <string>Звуки тише</string>
   </property>
  </action>
  <action name="switchSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Звуки включены</string>
   </property>
  </action>
  <action name="alertUp">
   <property name="text">
    <string>Уведомления громче</string>
   </property>
  </action>
  <action name="alertLow">
   <property name="text">
    <string>Уведомления тише</string>
   </property>
  </action>
  <action name="switchAlert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Уведомления включены</string>
   </property>
  </action>
  <action name="findOutput">
   <property name="text">
    <string>Текущий способ вывода</string>
   </property>
  </action>
  <action name="undo">
   <property name="text">
    <string>Отменить последнее действие</string>
   </property>
  </action>
  <action name="redo">
   <property name="text">
    <string>Вернуть последнее действие</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>enterText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
